main.go

```diff
@@ -25,11 +26,14 @@ func HealthCheckHandler(w http.ResponseWriter, r *http.Request) {
 	renderer.JSON(w, http.StatusOK, map[string]string{"status": "OK"})
 }
 
-func BuildHTTPMux(arbiterURL, contentStoreURL string) *http.ServeMux {
-	httpMux := http.NewServeMux()
-	httpMux.HandleFunc("/healthcheck", HealthCheckHandler)
+func BuildHTTPMux(arbiterURL, contentStoreURL string) *warp.ServeMux {
```

Comment:
It's probably cleaner for this to return a `http.Handler` as opposed to a pointer to a specific type.  All we need in the caller is something that implements that interface.